webapp/src/css/inbox.less

```diff
@@ -1151,13 +1172,12 @@ a.fa:hover {
 .analytics {
   .content {
     .inner {
-      /* allow the scrollable content to grow to full width */
```

Comment:
Did you test the scroll in these places?
- the aggregate list 
- the aggregates on the right side (the content)the cards below the main section 
Both work fine when the filter is closed and opened.
Also, please check the scroll inside the sidebar filter. 
Devices: Desktop, mobile, tablet.